hw/xbox/dsp/dsp_cpu.c

```diff
@@ -420,23 +431,29 @@ void dsp56k_reset_cpu(dsp_core_t* dsp)
     dsp->disasm_prev_inst_pc = 0xFFFFFFFF;
 }
 
-static OpcodeEntry lookup_opcode(uint32_t op) {
-    OpcodeEntry r = {0};
+static OpcodeEntry* opcode_identify(uint32_t op)
```

Comment:
The existing naming scheme is `activity_object` so `identify_opcode`.

The name is fine. Personally, I prefer exposing *some* inner workings in the name to warn users about characteristics.
`identify_opcode` sounds like magical instant-return oracle (choice of implementation might allow this: like the array lookup wrapper you have created).
`search_opcode` sounds like it will take a while (and is literally what this does, we have no implementation choice to avoid searching).

Reply:
oky, changed it to `search_opcode`